packages/esm-reports-app/src/index.ts

```diff
@@ -30,6 +30,11 @@ export function startupApp() {
       path: `${window.spaBase}/reports/scheduled-overview`,
       parent: `${window.spaBase}/reports`,
     },
+    {
+      title: () => Promise.resolve(window.i18next.t('reportsDataOverview', 'Reports Data Overview')),
```

Comment:
I realize that this is following the pattern of the other entries here, but, in point of fact, this won't actually translate correctly. The call should be `window.i18next.t('reportsDataOverview', { ns: '@openmrs/esm-reports-app', default: 'Reports Data Overview' })`